internal/core/docker_host.go

```diff
@@ -262,6 +270,16 @@ func dockerHostFromContext(ctx context.Context) (string, error) {
 	return "", ErrDockerSocketNotSetInContext
 }
 
+// dockerHostFromContext returns the docker host from the Go context, if it's not empty
+func dockerHostFromDockerContext(ctx context.Context) (string, error) {
```

Comment:
question: is this method needed, seems like its just enforcing that the returned string is blank on error which I would expected to be the case anyway?

Reply:
I think this method is basically passing an unused context to satisfy the "contract" in the above slice of functions. I think we can remove it and simply define an anonymous function for that.